<launch>
    <node
        pkg="yj_controller"
        exec="yj_controller_node"
        name="MotorControl"
        output="screen">
        <param name="K_pos" value="[3.0, 3.0, 3.0]"/>
        <param name="K_vel" value="[3.0, 3.0, 3.0]"/>
        <param name="K_Rot" value="[0.3, 0.3, 0.3]"/>
        <param name="K_omega" value="[0.2, 0.2, 0.1]"/>
        <param name="m_b" value="2.3643"/>
        <param name="J_bx" value="0.023847"/>
        <param name="J_by" value="0.023950"/>
        <param name="J_bz" value="0.044000"/>
        <param name="g" value="9.81"/>
        <param name="T_max" value="34.0"/>
        <param name="Tau_max_x" value="2.97"/>
        <param name="Tau_max_y" value="2.97"/>
        <param name="Tau_max_z" value="0.274"/>
        <param name="d" value="0.174"/>
        <param name="c_tau_f" value="0.016"/>
        <param name="t_max" value="8.54858"/>
        <param name="dz_takeoff" value="0.5"/>
        <param name="dxy" value="0.1"/>
        <param name="dz" value="0.1"/>
        <param name="drot" value="5.0"/>

        <!-- t_max = 8.54858 N, T_max = 34.19432 N, tau_max = 0.136777 N.m, Tau_max = [2.9749, 2.9749, 0.273554]-->
    </node>
</launch>